datapackage_pipelines_assembler/schemas/assembler_spec_schema.json

```diff
@@ -8,9 +8,21 @@
         "owner": {"type": ["string", "null"]},
         "ownerid": {"type": "string"},
         "dataset": {"type": "string"},
-        "version": {"type": "integer"}
+        "version": {"type": "integer"},
```

Comment:
This is the json schema for validating the source specs - not related to elasticsearch...

Reply:
This is where you need to change the schema:
https://github.com/datahq/assembler/blob/master/datapackage_pipelines_assembler/processors/dump_to_s3.py#L18

:)